src/rdkafka_cgrp.c

```diff
@@ -5173,10 +5202,45 @@ rd_kafka_cgrp_subscription_set(rd_kafka_cgrp_t *rkcg,
                                rd_kafka_topic_partition_list_t *rktparlist) {
         int32_t new_subscription_version =
             rd_atomic32_add(&rkcg->rkcg_subscription_version, 1);
```

Comment:
This can be done at the end of the function and directly returned without the variable